apps/01PigLatin.js

```diff
@@ -7,8 +7,37 @@ prompt.start();
 
 function pigLatin(word) {
 
-    // Your code here
+var firstLetter = word[0];
+return word.replace(firstLetter, '') + firstLetter + 'ay';
+console.log( 'crazy'.indexOf('a') );
 
+console.log( 'crazy'.indexOf('e') );
+var word = 'crazy';
+var vowelIndex = -1; // Set it to assume there are no vowels
+
+if ( ( word.indexOf('a') > -1 && word.indexOf('a') < vowelIndex ) || vowelIndex === -1 ) {
+    vowelIndex = word.indexOf('a');
+}
+
+if ( ( word.indexOf('e') > -1 && word.indexOf('e') < vowelIndex ) || vowelIndex === -1 ) {
+    vowelIndex = word.indexOf('e');
+}
+
+if ( ( word.indexOf('i') > -1 && word.indexOf('i') < vowelIndex ) || vowelIndex === -1 ) {
+    vowelIndex = word.indexOf('i');
+}
+
+console.log( 'hello'.slice(0, 2) );
+var firstPart = word.slice(0, vowelIndex);
+var restWord = word.slice(vowelIndex, word.length);
+
+return restWord + firstPart + 'ay';
+
+console.log( pigLatin('egg') );
+if vowelIndex(0)
+  return word + 'yay'
+
+console.log( 'HeLlO'.toLowerCase() );
```

Comment:
I'm currently trying to push my changes to git. My machine says its working but I don't see the changes here.